location-service/src/main/java/uk/gov/cshr/locationservice/service/GoogleService.java

```diff
@@ -129,4 +131,10 @@ public Coordinates getCoordinates(double latitude, double longitude) {
         return new Coordinates(latitude, longitude, ukNuts != null ? ukNuts.getName() : null);
     }
 
+    @CacheEvict(value = "coordinates", allEntries = true)
+    @Scheduled(fixedDelay = Integer.MAX_VALUE)
```

Comment:
can this delay be externalized to support Ravi testing @ravigeda , put the value in application.yml so that Ravi can do the testing by passing a different value when booting the app by using -Dkey=timeinint

Reply:
how do you do that?